<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MyPrestations</string>
  </property>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>490</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBox" >
    <property name="geometry" >
     <rect>
      <x>7</x>
      <y>63</y>
      <width>377</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable" >
     <bool>true</bool>
    </property>
    <property name="modelColumn" >
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>377</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>108</x>
      <y>14</y>
      <width>138</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Dur√©e</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit" >
    <property name="geometry" >
     <rect>
      <x>108</x>
      <y>34</y>
      <width>138</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>8</x>
      <y>15</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit" >
    <property name="geometry" >
     <rect>
      <x>8</x>
      <y>35</y>
      <width>83</width>
      <height>19</height>
     </rect>
    </property>
    <property name="displayFormat" >
     <string>dd/MM/yyyy</string>
    </property>
    <property name="date" >
     <date>
      <year>2008</year>
      <month>8</month>
      <day>26</day>
     </date>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>470</y>
     <width>101</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>110 minutes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>470</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Total :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>680</x>
     <y>590</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3" >
   <property name="geometry" >
    <rect>
     <x>670</x>
     <y>20</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Nouveau projet</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>761</width>
     <height>411</height>
    </rect>
   </property>
   <property name="rowCount" >
    <number>0</number>
   </property>
   <property name="columnCount" >
    <number>3</number>
   </property>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QFrame" name="frame_2" >
   <property name="geometry" >
    <rect>
     <x>420</x>
     <y>490</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_6" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;&lt; Envoyer</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber" >
    <property name="geometry" >
     <rect>
      <x>122</x>
      <y>46</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint" >
     <bool>false</bool>
    </property>
    <property name="numDigits" >
     <number>1</number>
    </property>
    <property name="mode" >
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0" >
     <double>2.000000000000000</double>
    </property>
    <property name="intValue" stdset="0" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2" >
    <property name="geometry" >
     <rect>
      <x>102</x>
      <y>46</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint" >
     <bool>false</bool>
    </property>
    <property name="numDigits" >
     <number>1</number>
    </property>
    <property name="mode" >
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0" >
     <double>2.000000000000000</double>
    </property>
    <property name="intValue" stdset="0" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3" >
    <property name="geometry" >
     <rect>
      <x>76</x>
      <y>46</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint" >
     <bool>false</bool>
    </property>
    <property name="numDigits" >
     <number>1</number>
    </property>
    <property name="mode" >
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0" >
     <double>2.000000000000000</double>
    </property>
    <property name="intValue" stdset="0" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_4" >
    <property name="geometry" >
     <rect>
      <x>56</x>
      <y>46</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint" >
     <bool>false</bool>
    </property>
    <property name="numDigits" >
     <number>1</number>
    </property>
    <property name="mode" >
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0" >
     <double>2.000000000000000</double>
    </property>
    <property name="intValue" stdset="0" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_5" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>46</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint" >
     <bool>false</bool>
    </property>
    <property name="numDigits" >
     <number>1</number>
    </property>
    <property name="mode" >
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0" >
     <double>2.000000000000000</double>
    </property>
    <property name="intValue" stdset="0" >
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_6" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>46</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <property name="smallDecimalPoint" >
     <bool>false</bool>
    </property>
    <property name="numDigits" >
     <number>1</number>
    </property>
    <property name="mode" >
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle" >
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0" >
     <double>2.000000000000000</double>
    </property>
    <property name="intValue" stdset="0" >
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_5" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>790</x>
     <y>520</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Actu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
